modules/build-raptor-action/src/build-raptor-action.ts

```diff
@@ -17,8 +19,28 @@ interface Options {
   printPassing: boolean
 }
 
-function getEnv(envVarName: 'GITHUB_SHA' | 'CI') {
-  return process.env[envVarName] // eslint-disable-line no-process-env
+async function getLatestMainMergedPRNumber(): Promise<number> {
+  const response = await axios.get(
+    `https://api.github.com/repos/moojo-tech/antelope/pulls?state=closed&base=main&sort=updated&direction=desc`,
```

Comment:
we could generalize this function, such that we read the ORG, REPO name, and the name of the base branch